exercises/phone-number/canonical-data.json

```diff
@@ -84,6 +84,21 @@
         "error": "more than 11 digits"
       }
     },
+    {
+      "uuid": "4a1509b7-8953-4eec-981b-c483358ff531",
+      "reimplements": "c6a5f007-895a-4fc5-90bc-a7e70f9b5cad",
+      "description": "invalid when more than 11 digits",
+      "comments": [
+        "Share incorrect number error instead of using unique error."
+      ],
+      "property": "clean",
+      "input": {
+        "phrase": "321234567890"
+      },
+      "expected": {
+        "error": "incorrect number of digits"
```

Comment:
We could improve this error message by communicating the required number of digits, rather than just saying it is wrong.

Reply:
We could reimplement them to say something along the lines of "phone number must be 10-11 digits long" or do separate errors for < 10 and > 11.